docs/release-notes.txt

```diff
@@ -4,6 +4,112 @@
  Release Notes
 ###############
 
+**************
+ Version 0.17
+**************
+
+Version 0.17.0
+==============
+
+**Release Date:** September 28, 2021
+
+**Breaking Changes**
+
+-  Remove ``--auto-bind-mount`` support from ``det deploy local``. The new ``--auto-work-dir``
+   feature should be a strictly better experience. Users who depended on the ``shared_fs`` directory
+   created by ``--auto-bind-mount`` can implement the same behavior by calling ``det deploy local
+   cluster_up`` with a ``--master-config-path`` pointing to a ``master.yaml`` file containing the
+   following text:
+
+   .. code:: yaml
+
+      task_container_defaults:
+        bind_mounts:
+          container_path: ./shared_fs
+          host_path: /path/to/your/HOME/dir
+
+-  This version of ``det deploy`` will not be able to deploy previous versions of Determined. If you
+   need to deploy an older version, please use a matching version of the ``determined`` package.
+
+-  Include ``maxval`` in ``int``-type hyperparameter ranges. Previously, the docs said that the
+   endpoints of the hyperparameter were both inclusive, but in reality the upper limit ``maxval``
+   was never actually selected.
+
+   The reproducibility of hyperparameter selection may differ between Determined v0.16.5 and v0.17.0
+   for hyperparameter searches containing ``int``-type hyperparameters as a result of this fix.
+   However, the reproducibility of model training for any given set of hyperparameters should be
+   unaffected.
+
+-  API endpoints no longer return the start times of workloads (training, validation, and
+   checkpoints). This is part of a longer move to model metrics and workloads separately as part of
+   the upcoming generic API.
+
+-  ``det master config`` now outputs YAML instead of JSON by default. To obtain the old behavior,
+   run ``det master config -o json``.
+
+**New Features**
+
+-  Support configuring an idle timeout for TensorBoards and notebooks.
```

Comment:
```suggestion
-  Notebook and TensorBoard: Supports a configurable timeout field idle_timeout that will automatically shut down the Notebook and TensorBoard instances after the desired timeout period. A Notebook is considered to be idle if no kernels and terminals are running and no network traffic is going to the server. A TensorBoard is considered to be idle if there is no network traffic going to the server Note that if you open a Notebook file it might open a kernel for you, and the kernels and the terminals will not be shut down automatically. You need to manually shut down the kernels to make idle timeout effective.
```